views/tracks.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="trackrow">
+    {{#if tracks}}
+        {{#each tracks}}
+        <p>{{this.preview_url}}</p>
+            <h1>{{this.artists.[0].name}}</h1>
```

Comment:
this might potentially break, in case that the artist at pos 0 is the "wrong" artist...

Reply:
in case of a feature or something like this. better to either -> display all of them, or send the "correct" artist from the server based on the actual artistId used in the search.